__tests__/hooks/use-display-currency.spec.tsx

```diff
@@ -0,0 +1,220 @@
+          defaultAccount: {
+            __typename: "Account",
+            id: "84b26b88-89b0-5c6f-9d3d-fbead08f79d8",
+            realtimePrice: {
+              btcSatPrice: {
+                base: 24015009766,
+                offset: 12,
+                currencyUnit: "USDCENT",
+                __typename: "PriceOfOneSat",
+              },
+              denominatorCurrency: "NGN",
+              id: "67b6e1d2-04c8-509a-abbd-b1cab08575d5",
+              timestamp: 1677184189,
+              usdCentPrice: {
+                base: 100000000,
+                offset: 6,
+                currencyUnit: "USDCENT",
+                __typename: "PriceOfOneUsdCent",
+              },
+              __typename: "RealtimePrice",
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    request: {
+      query: CurrencyListDocument,
+    },
+    result: {
+      data: {
+        currencyList: [
+          {
+            flag: "🇳🇬",
+            id: "NGN",
+            name: "Nigerian Naira",
+            symbol: "₦",
+            fractionDigits: 2,
+            __typename: "Currency",
+          },
+        ],
+      },
+    },
+  },
+]
+
+const mocksJpy = [
+  {
+    request: {
+      query: RealtimePriceDocument,
+    },
+    result: {
+      data: {
+        me: {
+          __typename: "User",
+          id: "70df9822-efe0-419c-b864-c9efa99872ea",
+          defaultAccount: {
+            __typename: "Account",
+            id: "84b26b88-89b0-5c6f-9d3d-fbead08f79d8",
+            realtimePrice: {
+              btcSatPrice: {
+                base: 24015009766,
+                offset: 12,
+                __typename: "PriceOfOneSat",
+              },
+              denominatorCurrency: "JPY",
+              id: "67b6e1d2-04c8-509a-abbd-b1cab08575d5",
+              timestamp: 1677184189,
+              usdCentPrice: {
+                base: 100000000,
+                offset: 6,
+                __typename: "PriceOfOneUsdCent",
+              },
+              __typename: "RealtimePrice",
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    request: {
+      query: CurrencyListDocument,
+    },
+    result: {
+      data: {
+        currencyList: [
+          {
+            flag: "",
+            id: "JPY",
+            name: "Japanese Yen",
+            symbol: "¥",
+            fractionDigits: 0,
+            __typename: "Currency",
+          },
+        ],
+      },
+    },
+  },
+]
+
+/* eslint-disable react/display-name */
+/* eslint @typescript-eslint/ban-ts-comment: "off" */
+const wrapper =
+  // @ts-ignore-next-line no-implicit-any error
+
+
+    (mocks) =>
+    ({ children }: PropsWithChildren) =>
+      (
+        <IsAuthedContextProvider value={true}>
+          <MockedProvider mocks={mocks}>{children}</MockedProvider>
+        </IsAuthedContextProvider>
+      )
+
+describe("usePriceConversion", () => {
+  describe("testing moneyAmountToMajorUnitOrSats", () => {
```

Comment:
I'm not sure why we have nested `describe` functions when we can have one??

Reply:
nested describe should not be an issue when it's bringing useful context, you can look into the backend, it's a common practice there